live-examples/js-examples/typedarray/typedarray-findlastindex.js

```diff
@@ -0,0 +1,8 @@
+function isNegative(element, index, array) {
```

Comment:
I don't know the convention here, so this is more of a question. But shouldn't we have (eventually with a comment):
```suggestion
function isNegative(element) {
```

Reply:
Thanks @teoli2003. I'm going to instead comment the unused elements. They are almost never used, but I think it is worth making the point, even here, that they are supplied. If you strongly disagree I can strip them out - they are almost never used.

```suggestion
function isNegative(element /*, index, array */) {
```

Reply:
No, I think this is the good way of doing it.